project_stock/models/project_project.py

```diff
@@ -34,13 +34,15 @@ class ProjectProject(models.Model):
 
     @api.onchange("picking_type_id")
     def _onchange_picking_type_id(self):
-        self.location_id = self.picking_type_id.default_location_src_id.id
-        self.location_dest_id = self.picking_type_id.default_location_dest_id.id
+        self.location_id = self.picking_type_id.default_location_src_id
```

Comment:
No need to change this, or is there any side effect I don't know?

Reply:
There is no side effect, but I think the right thing to do is not to define the id (as it happens in other odoo core examples).

Reply:
I think Odoo doesn't have any guideline about that, so the normal trend is to keep minimum the diff.